### DISCUSSION TITLE
What could be causing this?

### DISCUSSION THREAD

**Discussion by feixia5712** [ccfos/nightingale#1244] [General]
`
Nov  1 09:49:05 yeyingv5 n9e-server: 2022-11-01 09:49:05.780553 WARNING writer/writer.go:85 push data with remote write request got status code: 400, response body: duplicate sample for timestamp
Nov  1 09:49:15 yeyingv5 prometheus: ts=2022-11-01T01:49:15.737Z caller=write_handler.go:57 level=error component=web msg="Out of order sample from remote write" err="duplicate sample for timestamp"
Nov  1 09:49:15 yeyingv5 n9e-server: 2022-11-01 09:49:15.737931 WARNING writer/writer.go:85 push data with remote write request got status code: 400, response body: duplicate sample for timestamp
Nov  1 09:49:25 yeyingv5 prometheus: ts=2022-11-01T01:49:25.761Z caller=write_handler.go:57 level=error component=web msg="Out of order sample from remote write" err="duplicate sample for timestamp"
Nov  1 09:49:25 yeyingv5 n9e-server: 2022-11-01 09:49:25.761508 WARNING writer/writer.go:85 push data with remote write request got status code: 400, response body: duplicate sample for timestamp
Nov  1 09:49:35 yeyingv5 prometheus: ts=2022-11-01T01:49:35.769Z caller=write_handler.go:57 level=error component=web msg="Out of order sample from remote write" err="duplicate sample for timestamp"
Nov  1 09:49:35 yeyingv5 n9e-server: 2022-11-01 09:49:35.769438 WARNING writer/writer.go:85 push data with remote write request got status code: 400, response body: duplicate sample for timestamp`

是某台agent时间不准确导致的码，有没有什么办法找到是哪台机器

**Comment by UlricQin** [comment]
server.conf 里有个配置：
```
[WriterOpt]
# queue channel count
QueueCount = 100
# queue max size
QueueMaxSize = 200000
# once pop samples number from queue
QueuePopSize = 2000
```
把QueuePopSize设置为1，再观察日志

**Reply by feixia5712** [reply]
配置修改了
![image](https://user-images.githubusercontent.com/12670758/199185150-dac88bcd-7f4c-40e4-a85d-fb66dc09e77f.png)
重启之后还是不行
![image](https://user-images.githubusercontent.com/12670758/199185013-594ce2fa-1887-45b8-a9e1-c328295e0a00.png)

**Comment by UlricQin** [comment]
不是说这样改就可以解决了。是说这样改了之后打印的日志就是有问题的机器

**Comment by feixia5712** [comment]
目前的版本
![image](https://user-images.githubusercontent.com/12670758/199186384-87bad2de-a614-429f-a478-397e028ba1b6.png)
改了之后日志还是这样，是不是我的版本太老了，改了也看不出？
![image](https://user-images.githubusercontent.com/12670758/199186438-cfb708b1-bcc6-49b8-a1e6-73e8da221742.png)

**Comment by UlricQin** [comment]
版本太老了。最新版本是：https://github.com/ccfos/nightingale/releases/tag/v5.14.0

**Comment by feixia5712** [comment]
好的，我慢慢更新升级到最新版本再看看，谢谢
